compendium/DeclarativeServices/src/manager/ConfigurationNotifier.cpp

```diff
@@ -127,6 +126,14 @@ namespace cppmicroservices
                 auto iter = listenersMapHandle->find(pid);
                 if (iter != listenersMapHandle->end())
                 {
+                    // Make sure the properties does not contain any dynamic targets
+                    auto tokenMapPtr = iter->second;
+                    for (auto const& tokenEntry : (*tokenMapPtr))
+                    {
+                        auto listener = tokenEntry.second;
+                        VerifyProperties(properties, listener.mgr);               
```

Comment:
this method only logs and doesn't really verify. I recommend renaming the method. Also, I question the usefulness of this method; is all this processing necessary to maybe log an error?